docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
 
   tyk-gateway:
     image: tykio/tyk-gateway:${TYK_VERSION:-v5.0.0-amd64}
+    platform: linux/amd64
```

Comment:
Not sure if this is needed anymore, but at some point in the past we ran into problems where the arm64 image of Gateway was running instead of the amd64 version.  Until the plugin compiler works on arm64 we need to force gateway to run in amd64 compatibility mode on ARM-based Macs.

Reply:
I think with 503 or 504 will add the amr64 compiler image. I know I had to do that in the docker compose at some point.

Reply:
5.0.3 and 5.1.1, released now.

Reply:
I've found this field to do basically nothing.  These issues affirm this bug in docker compose:
https://github.com/docker/for-mac/issues/5364
https://github.com/docker/compose/pull/8147